### DISCUSSION TITLE
timescale of months

### DISCUSSION THREAD

**Discussion by FADHLyemen** [theislab/scvelo#496] [Q&A]
I got scRNA-seq from samples treated for one month and other treated for 10 months. can I use scvelo to plot the velocity between cells. what is the max timescale of hours, scvelo can have?

"RNA velocity is a high-dimensional vector that predicts the future state of individual cells on a timescale of hours."
Using the below figure, I am trying to hypothesize that cells at the most left trigger/respond first to the treatment. Do you have suggestions for other tools?
![image](https://user-images.githubusercontent.com/23288387/121196478-83d16d80-c83e-11eb-9b00-3f23f6b6d63d.png)

**Comment by WeilerP** [comment]
@FADHLyemen, just to make sure, I understood correctly: You have data at time points *t_1 = 1*month and *t_2 = 10*months?

Defining a maximum number of hours for RNA velocity to still work is, in general, not possible. This depends on RNA half-life and if/how the data at your given time points overlap (in high dimensional space). When looking at a difference of several months, RNA velocity will not work.
You could try [Waddington OT](https://broadinstitute.github.io/wot/) (WOT) which restricts a linear relationship between time points. I am not sure at all how well this will work in your case. If you are interested, you can use the `WOTKernel` on `cellrank@dev` to estimate fate probabilities, etc. using [CellRank](https://github.com/theislab/cellrank). Maybe [DPT](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.dpt.html) combined with CellRank could give you some desired info as well. But again, not sure at all how well this works in your case.

Regarding you last question. The figure displays the estimated cell differentiation. I'm not sure about your hypothesis (and why you think so). You can, of course, argue that since *Ductal* cells exist prior to, let's say, *Beta* cells, the former reacts first to the treatment. But this is a different interpretation of "reacting first" compared to starting out with *Ductal* and *Beta* cells and checking which cell type responds to the treatment first. The treatment could also only target certain cell types, right?

**Reply by FADHLyemen** [reply]
Thank you, @WeilerP 
 used scCODA to identify which cell types {cell A} response to the treatment. My design is some sample with short period treatment{< six months} and other long exposure treatment{>6 months}. when I plot the PAGA diagram for only short exposure samples, I got cell A at the most left of the graph, then I said ok maybe the support the sccoda finding. But when I read you above comment, I got that getting cell type first exist did not mean it response to the treatment first.
